funwithphysics/src/Components/Thermodynamics/Topics/Calculator.js

```diff
@@ -226,6 +226,7 @@ function Calculator({ match }) {
       </div>
       <div className="Calculator__details">
         <p>{details.details}</p>
+        For more Information visit <a href="https://en.wikipedia.org/wiki/Kinetic_theory_of_gases" target="_blank" rel="noreferrer">Wikipedia</a>
       </div>
```

Comment:
U have already added the link along with the topic details then why u again added here?

Reply:
ok i will remove it

Reply:
removed !!